src/lib/server/server-provider.ts

```diff
@@ -0,0 +1,140 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+import {
+  InjectionToken,    // tslint:disable-line:no-unused-variable
+  ComponentRef,      // tslint:disable-line:no-unused-variable
+} from '@angular/core';
+import {DOCUMENT} from '@angular/common';
+import {BEFORE_APP_SERIALIZED} from '@angular/platform-server';
+
+import {
+  BreakPoint,
+  BREAKPOINTS,
+  CLASS_NAME,
+  MatchMedia,
+  ServerStylesheet,
+  SERVER_TOKEN,
+  ServerMatchMedia
+} from '@angular/flex-layout';
+
+let UNIQUE_CLASS = 0;
```

Comment:
`UNIQUE_CLASS` shouldn't be caps case because it's not constant. I typically call this `nextId` in material ([example](https://github.com/angular/material2/blob/3aceb7361cc34ad987f7b1ca39339d3203248341/src/cdk/accordion/accordion-item.ts#L24))

Reply:
Fixed